src/components/Editor/components/MyWork/index.js

```diff
@@ -67,6 +67,9 @@ export default class MyWork extends Component {
         }
         if (event.ctrlKey) {
             keys.push("Ctrl");
+            if (event.key == " ") {
```

Comment:
We should inform the user how to insert space in the math field, so maybe it should also be attached to some button?

Reply:
as above